<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Главная</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QTabWidget" name="tw_main">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_main">
       <attribute name="title">
        <string>Главная</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="5,2">
        <item>
         <widget class="QListWidget" name="lW_client"/>
        </item>
        <item>
         <widget class="QTableView" name="tvw_requestQueue"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_in">
       <attribute name="title">
        <string>Поступление</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="10,1">
          <item>
           <widget class="QComboBox" name="cb_vendors"/>
          </item>
          <item>
           <widget class="QCheckBox" name="chb_listenClipboard">
            <property name="text">
             <string>Слушать</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="tvw_inProc">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="tvw_inDb">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="btn_processIn">
            <property name="text">
             <string>Провести</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_addRecord">
            <property name="text">
             <string>Добавить</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_out">
       <attribute name="title">
        <string>Отпуск</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTableView" name="tvw_outWh">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="btn_addOut">
            <property name="text">
             <string>Добавить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_deleteOut">
            <property name="text">
             <string>Удалить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_processOut">
            <property name="text">
             <string>Провести</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="tvw_out">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_goods">
       <attribute name="title">
        <string>Товар</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QTableView" name="tvw_report_goods"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="3,1,0,6">
          <item>
           <widget class="QComboBox" name="cb_depots"/>
          </item>
          <item>
           <widget class="QPushButton" name="btn_getDepotList">
            <property name="text">
             <string>Загрузить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_selectRetail">
            <property name="text">
             <string>Выбрать</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_registry">
       <attribute name="title">
        <string>Реестр</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QTableView" name="tvw_report_regisry"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="3,0,1,6">
          <item>
           <widget class="QComboBox" name="cb_registry_types"/>
          </item>
          <item>
           <widget class="QPushButton" name="btn_getRegistryTypes">
            <property name="text">
             <string>Загрузить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_selectRegistry">
            <property name="text">
             <string>Выбрать</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_control">
       <attribute name="title">
        <string>Управление</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="btn_registerProduct">
            <property name="text">
             <string>Зарегистрировать продукт</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_alterProductData">
            <property name="text">
             <string>Изменить данные продукта</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>479</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>940</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="act_settings"/>
    <addaction name="separator"/>
    <addaction name="act_exit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="act_exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="act_settings">
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
